src/models/neural-clean-conversation.ts

```diff
@@ -0,0 +1,36 @@
+import Message from './message';
+import Attributes from './attributes';
+import File from './file';
+
+export default class NeuralCleanConversation extends Message {
+  conversation?: string;
+  modelVersion?: string;
+
+  extractImages(): Promise<File[]> {
+    const f: File[] = [];
+    if (this.conversation) {
+      const regex = /[(']cid:(.)*[)']/g;
+      const fileIds = this.conversation.match(regex);
+      if (fileIds) {
+        fileIds.forEach(async id => {
+          const parsedId = id.substring(5, id.length - 1);
```

Comment:
would be nice to have a comment about how id looks like

Reply:
Good idea.... I should probably have better comments in general